examples/text-generation/utils.py

```diff
@@ -138,7 +138,7 @@ def setup_device(args):
     if args.device == "hpu":
         import habana_frameworks.torch.core as htcore
 
-        if args.fp8:
+        if args.quant_config:
```

Comment:
This is true also when performing a measurement, was it tested?

Reply:
yes